src/server.c

```diff
@@ -2911,12 +2921,25 @@ void initServer(void) {
         serverPanic("Scripting engine manager initialization failed, check the server logs.");
     }
 
+    // Since we initialized the scripting engine manager, we need to ensure
+    // that commands with `CMD_NOSCRIPT` flag are not allowed to run in scripts.
+    //
```

Comment:
nit - let's keep the c style comments for now. I don't think we have closed on the discussion of using c++ style yet.

Reply:
We already have many of them and clang-format doesn't say anything, but sure the C89 style is our main style.

```suggestion
    /* Since we initialized the scripting engine manager, we need to ensure that
     * commands with `CMD_NOSCRIPT` flag are not allowed to run in scripts. */
```

Reply:
Sorry, I spent the last 8 years coding in C++ and my muscle memory wrote this code :smile:

Reply:
This is valid C since 1999. It's just our old style convention.